pages/ethics-tree.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Ethics Decision Tree
+description: When to apply for an ethics approval
```

Comment:
I'm not 100% sure about the title and description: @LizGuzRam if you have any ideas, please suggest!

Reply:
oh yes yes, I can add a little bit of intro and the link to the IRBs pages if that's ok with you? :)

Reply:
That sounds good to me!

Reply:
OK added the intro and some links to the IRBs 👍

Reply:
Also, changed the image, cause I made the font a bit bigger, to make it more easy to read. 
I think I'm happy with the way it looks now. 
@nsunami do you want to do a final check? (although I cannot assign you as a reviewer)

Reply:
Graet, thanks! Let me check.